### PR TITLE
OSAC-260 - Update Resource for Database activities to match FIGMA des…

### PR DESCRIPTION
**Pull request by mateisimionescu**
…igns
Description
StudioWeb activities for Database package
How to test
Open Activity package in Studio
Jira
[OSAC-260]((https://uipath.atlassian.net/browse/OSAC-260))
Label:
24S

### CODE REVIEW

File: Activities/Database/UiPath.Database.Activities.Design/UiPath.Database.Activities.Design.csproj [hunk 1]
```diff
@@ -11,7 +11,7 @@
     <PackageReference Include="System.Activities" />
     <PackageReference Include="UiPath.Workflow" />
     <PackageReference Include="System.Activities.Core.Presentation" />
-    <PackageReference Include="Oracle.ManagedDataAccess.Core" Version="3.21.1" PrivateAssets="All" />
+	<PackageReference Include="Oracle.ManagedDataAccess.Core" Version="3.21.1" PrivateAssets="All" />
```
Comment (alcxs) [inline Activities/Database/UiPath.Database.Activities.Design/UiPath.Database.Activities.Design.csproj:14]: why the spacing? please remove

File: Activities/Database/UiPath.Database.Activities.Design/UiPath.Database.Activities.Design.csproj [hunk 2]
```diff
@@ -30,6 +30,8 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="System.Resources.Extensions" Version="5.0.0" />
+    <PackageReference Include="UiPath.Activities.Api" />
+	<PackageReference Include="UiPath.Activities.Contracts" />
```
Comment (alcxs) [inline Activities/Database/UiPath.Database.Activities.Design/UiPath.Database.Activities.Design.csproj:34]: remove extra spacing

File: Activities/Database/UiPath.Database.Activities.Design/DesignerMetadata.cs [hunk 3]
```diff
@@ -7,11 +7,34 @@
 using System.Windows.Markup;
 using UiPath.Activities.Presentation.Editors;
 using UiPath.Database.Activities.Design.Properties;
+using UiPath.Studio.Activities.Api;
+
```
Comment (alcxs) [inline Activities/Database/UiPath.Database.Activities.Design/DesignerMetadata.cs:11]: remove this

**Review by alcxs — CHANGES_REQUESTED** [review]
small formatting issues, otherwise looks good

File: Activities/Database/UiPath.Database.Activities.Design/UiPath.Database.Activities.Design.csproj (hunk 2 above)
Comment (alcxs) [inline Activities/Database/UiPath.Database.Activities.Design/UiPath.Database.Activities.Design.csproj:34]: spacing please

File: Activities/Directory.build.targets [hunk 4]
```diff
@@ -12,8 +12,8 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Update="UiPath.Robot.Api" Version="21.6.0-beta.4360" PrivateAssets="All" />
-	  <PackageReference Update="UiPath.Activities.Api" Version="21.6.0-beta.4360" PrivateAssets="All" />
-    <PackageReference Update="UiPath.Activities.Contracts" Version="21.6.0-beta.4360" PrivateAssets="All" />
+	<PackageReference Update="UiPath.Activities.Api" Version="21.12.0-alpha.6604" PrivateAssets="All" />
```
Comment (alcxs) [inline Activities/Directory.build.targets:15]: spacing

**Review by alcxs — APPROVED** [review]
minor styling issues

**Review by ovidiuiliescu — APPROVED** [review]

**Pull request merged**